libminifi/src/ThreadedSchedulingAgent.cpp

```diff
@@ -47,15 +47,15 @@ void ThreadedSchedulingAgent::schedule(std::shared_ptr<core::Processor> processo
   if (configure_->get(Configure::nifi_administrative_yield_duration, yieldValue)) {
     core::TimeUnit unit;
     if (core::Property::StringToTime(yieldValue, admin_yield_duration_, unit) && core::Property::ConvertTimeUnitToMS(admin_yield_duration_, unit, admin_yield_duration_)) {
-      logger_->log_debug("nifi_administrative_yield_duration: [%ll] ms", admin_yield_duration_);
+      logger_->log_debug("nifi_administrative_yield_duration: [%lld] ms", admin_yield_duration_);
```

Comment:
Use `PRId64` from `<cinttypes>` for `int64_t`. `lld` is meant to be used for `long long int`, which is not guaranteed to be the same as `int64_t`. It is not the same on my platform.

The above applies to both changes in this file.

Reply:
fixed by da22085